examples/java/src/main/java/com/xebia/functional/xef/java/auto/Poems.java

```diff
@@ -0,0 +1,28 @@
+package com.xebia.functional.xef.java.auto;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+
+public class Poems {
+    public static class Poem {
+        public String title;
+        public String content;
+    }
+
+    public static void main(String[] args) throws ExecutionException, InterruptedException {
+        try (AIScope scope = new AIScope()) {
+            CompletableFuture<Poem> poem1 = scope.prompt("A short poem about the beauty of nature.", Poem.class);
+            CompletableFuture<Poem> poem2 = scope.prompt("A short poem about the power of technology.", Poem.class);
+            CompletableFuture<Poem> poem3 = scope.prompt("A short poem about the wisdom of artificial intelligence.", Poem.class);
+
+            String combinedPoems = String.format("%s\n\n%s\n\n%s", poem1.get().content, poem2.get().content, poem3.get().content);
+            String newPoemPrompt = "Write a new poem that combines ideas from the following themes: the beauty " +
+                    "of nature, the power of technology, and the wisdom of artificial intelligence. Here are some " +
+                    "examples of poems on these themes: " + combinedPoems;
+
+            scope.prompt(newPoemPrompt, Poem.class).
+                    thenAccept((poem) -> System.out.printf("New Poem:\n\n" + poem.content))
```

Comment:
Remove the parentheses around the "poem" parameter

Reply:
Fixed